tools/buildsys/src/builder.rs

```diff
@@ -161,7 +184,7 @@ fn build(target: &str, build_args: &str, tag: &str, output: &str) -> Result<()>
     ));
 
     let create = args(format!("create --name {tag} {tag} true", tag = tag));
-    let cp = args(format!("cp {}:/output/. {}", tag, output));
+    let cp = args(format!("cp {}:/output/. {}", tag, build_dir.display()));
```

Comment:
Would it be better to use rust instead of cp to do a copy here?

Reply:
This is a `docker cp` rather than a regular `cp`.

Reply:
oh yeah